images/scripts/fedora.setup

```diff
@@ -213,11 +213,9 @@ dnf install -y $selinux_pkgs
 # for c-podman tests
 /var/lib/testvm/podman-images.setup
 
-# pre-install cockpit/ws container for /ws-container scenario; we only need that on the image which the container is based on
+# pre-install cockpit/ws container for /ws-container scenario
 CF=https://raw.githubusercontent.com/cockpit-project/cockpit/main/containers/ws/Dockerfile
-if curl --silent --show-error --fail "$CF" | grep -q "^FROM.*fedora:$VERSION_ID"; then
-    podman pull quay.io/cockpit/ws:latest
-fi
+podman pull quay.io/cockpit/ws:latest
```

Comment:
Does this interfere with coreos? This also has FEDORA has ID.

Reply:
No, it doesn't interfere. fedora-coreos has its own .setup, only fedora-{39,40,41} use this script.